pkg/action/dependency.go

```diff
@@ -48,25 +55,28 @@ func NewDependency() *Dependency {
 }
 
 // List executes 'helm dependency list'.
-func (d *Dependency) List(chartpath string, out io.Writer) error {
```

Comment:
Functions that are part of the public interface can't have their name or signature changed in a minor version. Helm follows semantic versioning so breaking changes can only happen in major versions. What we typically do is add a new function with the new signature. If the old function is superseded we mark it as deprecated in a comment.

Reply:
Thanks for pointing that out! I recovered back the original signatures for the public functions in 3c47f3b3. Also, I've added a deprecation note in this one. Please if it should have a different structure let me know.